cms/src/App.vue

```diff
@@ -71,6 +72,17 @@ onBeforeMount(async () => {
 
             await loginRedirect();
         });
+
+        socket.on("changeRequestAck", (data: any) => {
+            if (data.ack == "rejected") {
+                useNotificationStore().addNotification({
+                    title: `Changed request: ${data.ack}`,
+                    description: data.message,
+                    state: "error",
+                    timer: 4000,
+                });
```

Comment:
This does not yet fully comply with the acceptance criteria from the ticket:

> Create a (toast) notification with the error, and also state that the content has been reverted to its previous state.

I think we need to make sure this notification is understandable for non-technical users. "Change request" for example is also a concept that's foreign to those people, so I don't think it's helpful if we mention that.

Suggestion:

Title: Saving changes to server failed
Description: Your recent request to save changes has failed. The changes have been reverted, both in your CMS and on the server. Error message: `data.message`

Not sure if this is too long, but hopefully this doesn't happen too often in production anyways. I do think it's helpful to note the actual error as it's fairly readable in most cases, and can anyways then be screenshotted/forwarded to a developer if people have questions.

Reply:
Changes made